sandbox/settings.py

```diff
@@ -296,6 +296,14 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
         "richie/homepage.html maincontent": {
             "name": _("Main content"),
             "plugins": ["LargeBannerPlugin", "SectionPlugin"],
+            "child_classes": {
+                "SectionPlugin": [
+                    "CoursePlugin",
+                    "OrganizationPlugin",
+                    "CategoryPlugin",
+                    "LinkPlugin",
```

Comment:
add person plugin?

Reply:
We need to add a little bit of styles for this plugin, this will be in a new different PR.

Reply:
:+1: